root/static/scripts/guess-case/MB/GuessCase/Main.js

```diff
@@ -28,7 +28,7 @@ class GuessCase {
   entities: {
     [entityType: string]: {
       guess: (string) => string,
-      sortname: (string) => string,
+      sortname: (string, typeId?: ?number) => string,
```

Comment:
I'm confused about this - seems only one guessSortName (artist) uses an extra parameter, and it's a boolean. That's why I removed this, because it seemed to be unused. What am I missing?